tools-local/ship-public-apis/ship-public-apis.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
since .NET 7 is not LTS, but .NET 6 is, I think `net6.0` would be a better choice, since it'll avoid the same warning for longer.